subcommand/controller/command.go

```diff
@@ -46,14 +47,18 @@ func (c *Command) init() {
 	c.flagSet.BoolVar(&c.flagEnableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	c.flagSet.StringVar(&c.flagWebhookTLSCertDir, "webhook-tls-cert-dir", "",
+		"Directory that contains the tls cert and key required to communicate with the MutatingWebhookConfiguration")
```

Comment:
> this is the directory with certs for the webhook itself, not for the controller to communicate with the webhook

The certs here are used by the Handler for the webhook, which does run inside the controller. The MutatingWebhook has the CA certificate, and the handler has the cert & key that are signed by that CA so that they can communicate with the Mutating Webhook.

> do the cert and key have to have special file names so that they'll be picked up by the webhook server?

Yup. When the kind of the secret is TLS secret (which is what the webhook-cert-manager creates), the cert is under the `tls.crt` and the key `tls.key`. The webhook manager defaults to looking for files with those names.